docs/client.md

```diff
@@ -27,6 +27,7 @@
 **Kind**: global class  
```

Comment:
This is going to collide with Michael's PR, since these files will be added to git ignore